documentation/index.html.js

```diff
@@ -857,9 +857,9 @@ Block
       constructed.
     </p>
     <p>
-      CoffeeScript functions also support
-      <a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/function*">ES6 generator functions</a>
-      through the <code>yield</code> keyword. There's no <code>function*(){}</code>
+      CoffeeScript also supports
+      <a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/function*">generator functions</a> and <a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/async_function">async functions</a>
+      through the <code>yield</code> and <code>await</code> keywords respectively. There's no <code>function*(){}</code> or <code>async function(){}</code>
       nonsense &mdash; a generator in CoffeeScript is simply a function that yields.
```

Comment:
`a generator in CoffeeScript is simply a function that yields, and an async function in CoffeeScript is simply a function that awaits.` And then remove the `Likewise` line below.